SqlServerLib/SQLServer.cpp

```diff
@@ -0,0 +1,135 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check it.
+
+// PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
+
+#include "pch.h"
+#include "SQLServer.h"
+
+
+SQLServer::SQLServer(const ConnectParams& connect_string)
+{
+	LOG = LOG->GetInstance();
+	Connect(connect_string);
+}
+
+SQLServer::~SQLServer()
+{
+	Disconnect();
+}
+
+
+bool SQLServer::Connect(const ConnectParams& connect_string)
+{
+	bool check_connection = m_connection.isConnected();
+
+	if (!check_connection)
+	{
+		params = connect_string;
+		std::string con = connect_string.server + "@" + connect_string.database;
+		m_connection.Connect(_TSA(con.c_str()), _TSA(connect_string.username.c_str()), _TSA(connect_string.password.c_str()), SA_SQLServer_Client);
+		LOG_INFO << "Connected to SQL Server...\n";
+	}
+
+	return m_connection.isConnected();
+}
+
+void SQLServer::InsertEmail(const Email& email)
+{
+	std::string table_name{ "Users" };
```

Comment:
It will be better if those strings will be constexpr, because we will not change them ever and there will be no need to create them every time when we get into this method.
As a plus, some of that strings you are using in another methods, so it's win/win situation here.